packages/v4/patternfly-docs/content/accessibility/progress/progress.md

```diff
@@ -0,0 +1,93 @@
+---
+id: Progress
+section: components
+---
+
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
+
+## Accessibility
+
+To implement an accessible PatternFly **progress** component:
+- Ensure a `title` or an `aria-label` are being used to give context to users. Additional context could also be provided via an `aria-labelledby` attribute mapping to the `id` of existing content on the page.
+- Ensure that a live region exists on the page through `aria-live` prior to updating the value of the progress component
+- Add a message to screen readers inside of the live region when the progress value updates, for example: <code class="ws-code">`Progress value is ${currentValue}%.`</code>
```

Comment:
This is coming out with 2 boxes around the code block. Maybe we don't need both `<code>` and the class?